src/AuthApp.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { useContext } from 'react';
+import { AppContext } from './AppContext';
+import { LoginForm } from './Components/LoginForm';
+import { App } from './App';
+
+export const AuthApp: React.FC = () => {
+  const { user } = useContext(AppContext);
+
+  if (user === 0) {
```

Comment:
Try to use logical operators more 
```suggestion
  if (!user) {
```